.env

```diff
@@ -1,3 +1,4 @@
 SURVEY_RESPONSE_SHEETNAME_AND_CELLRANGE='Survey Responses'
 TRIPS_SHEETID='1t2vrLeczcowJvpkiVkFu_yc1AnfMoYarvdc1uoZXsPo'
-TRIPS_CELLRANGE='Total Trip counts by Transport Mode!A4:B6'
\ No newline at end of file
+TRIPS_CELLRANGE='Total Trip counts by Transport Mode!A4:B6'
+SURVEY_CLOSING_DATE='2022-07-29T20:00:00.123+10:00'
```

Comment:
hey @hqtan,
well done, if I have to be very picky, github points out that there is no empty space at the end 😄